.deepsource.toml

```diff
@@ -0,0 +1,18 @@
+version = 1
```

Comment:
Please remove this config file and please submit a feature request for discussing it's addition.

We have static analyzers running on PRs, though, admittedly, they did not catch the issues found on this PR.

Reply:
Thanks! @s0undt3ch :sparkling_heart: 

Created the issue for the same!